deploy/olm-catalog/ibm-block-csi-operator-community/1.2.0/ibm-block-csi-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,679 @@
+    repository: https://github.com/IBM/ibm-block-csi-operator
+    support: IBM
+    alm-examples: >-
+      [
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "IBMBlockCSI",
+          "metadata": {
+            "name": "ibm-block-csi"
+          },
+          "spec": {
+            "controller": {
+              "repository": "ibmcom/ibm-block-csi-driver-controller",
+              "tag": "1.2.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "node": {
+              "repository": "ibmcom/ibm-block-csi-driver-node",
+              "tag": "1.2.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "sidecars": [
+              {
+                "name": "csi-node-driver-registrar",
+                "repository": "quay.io/k8scsi/csi-node-driver-registrar",
+                "tag": "v1.2.0",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-provisioner",
+                "repository": "quay.io/k8scsi/csi-provisioner",
+                "tag": "v1.4.0",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "csi-attacher",
+                "repository": "quay.io/k8scsi/csi-attacher",
+                "tag": "v1.2.1",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "livenessprobe",
+                "repository": "registry.redhat.io/openshift4/ose-csi-livenessprobe",
```

Comment:
for community, use the 2.1.0 `quay.io/k8scsi/csi-snapshotter` (which you previously put in the Red Hat CSV)